microcloud/test/suites/preseed.sh

```diff
@@ -56,6 +57,13 @@ EOF
     validate_system_microovn ${m}
   done
 
+  # Check that the created UPLINK network has the right DNS servers.
+  dns_addresses=$(lxc exec micro01 -- sh -c "lxc network get UPLINK dns.nameservers")
+  if [ "${dns_addresses}" != "10.1.123.1,8.8.8.8" ] ; then
+    echo "ERROR: UPLINK network has wrong DNS server addresses: ${dns_addresses}"
+    return 1
+  fi
+
```

Comment:
Since this config key affects the input/output of every interactive CLI test, can you please move this logic over here:
https://github.com/canonical/microcloud/blob/0c3009cf15e68b1f58726489f7c699c14e35e6b9/microcloud/test/includes/microcloud.sh#L198

That validator is called from https://github.com/canonical/microcloud/blob/0c3009cf15e68b1f58726489f7c699c14e35e6b9/microcloud/test/includes/microcloud.sh#L255

So you can add an argument to `validate_system_lxd` for the comma-delimited list of nameservers, and have it send the args down to `validate_system_lxd_ovn` which will check the result in LXD if the argument is set. This will just make it easier to write future tests for the CLI and be able to consistently validate the resulting cluster.